test/HealthChecks.UI.Core.Tests/UIHealthReportTests.cs

```diff
@@ -0,0 +1,20 @@
+namespace HealthChecks.UI.Core.Tests;
+public class UIHealthReportTests
```

Comment:
Please use file-scoped namespaces for new code.

Reply:
Am I missing something? Because it is file-scoped namespace?

**file-scoped**
```csharp
namespace HealthChecks.UI.Core.Tests;
public class UIHealthReportTests
{
} 
```
**not file-scoped**
```csharp
namespace HealthChecks.UI.Core.Tests 
{
    public class UIHealthReportTests
    {
    } 
}

```

Reply:
I was wrong.